index.html

```diff
@@ -716,389 +716,343 @@ <h2>Identifier</h2>
+  the <a>DID method</a> itself.
+        </p>
+      </section>
+      <section id="method-specific-parameters">
+        <h2>Method-Specific Syntax</h2>
+  
+        <p>
+  A <a>DID method</a> specification MUST further restrict the generic <a>DID</a>
+  syntax by defining its own <code>method-name</code> and its own
+  <code>method-specific-id</code> syntax. For more information, see Section
+  <a href="#methods"></a>.
+        </p>
+  
+      </section>
+      <section>
+        <h2>Normalization</h2>
+  
+        <p>
+  For the broadest interoperability, make <a>DID</a> normalization as simple and
+  universal as possible:
+        </p>
+        <ul>
+          <li>
+  The <a>DID scheme</a> name MUST be lowercase.
+          </li>
+  
+          <li>
+  The <a>DID method</a> name MUST be lowercase.
+          </li>
+  
+          <li>
+  Case sensitivity and normalization of the value of the
+  <code>method-specific-id</code> rule in Section
+  <a href="#generic-did-syntax"></a> MUST be defined by the governing
+  <a>DID method</a> specification.
+          </li>
+        </ul>
+      </section>
+  
+      <section>
+        <h2>Persistence</h2>
+  
+        <p>
+  A <a>DID</a> is expected to be persistent and immutable. That is, a <a>DID</a>
+  is bound exclusively and permanently to its one and only subject. Even after a
+  <a>DID</a> is deactivated, it is intended that it never be repurposed.
+        </p>
+  
+        <p>
+  Ideally, a <a>DID</a> would be a completely abstract decentralized identifier
+  (like a <a>UUID</a>) that could be bound to multiple underlying
+  <a>DID registries</a> over time, thus maintaining its persistence independent of
+  any particular system. However, registering the same identifier on multiple
+  <a>DID registries</a> introduces extremely hard entityship and
+  <a href="https://en.wikipedia.org/wiki/List_of_DNS_record_types%23SOA">start-of-authority</a>
+  (SOA) problems. It also greatly increases implementation complexity for
+  developers.
+        </p>
+  
+        <p>
+  To avoid these issues, it is RECOMMENDED that <a>DID method</a> specifications
+  only produce <a>DIDs</a> and <a>DID methods</a> bound to strong, stable
+  <a>DID registries</a> capable of making the highest level of commitment to
+  persistence of the <a>DID</a> and <a>DID method</a> over time.
+        </p>
+  
+        <p class="note">
+  Although not included in this version, future versions of this specification
+  might support a <a>DID document</a> <code>equivID</code> property to establish
+  verifiable equivalence relations between <a>DIDs</a> representing the same
+  subject on multiple <a>DID registries</a>. Such equivalence relations can
+  produce the practical equivalent of a single persistent abstract <a>DID</a>. For
+  more information, see Section <a href="#future-work"></a>.
+        </p>
+      </section>
+     </section>
 
     <section>
-      <h2>Normalization</h2>
+      <h3>DID URLs</h3>
```

Comment:
```suggestion
      <h3>DID URL Syntax</h3>
```

Changing to match title change above for "DID Syntax".

Reply:
Done in https://github.com/w3c/did-core/pull/212/commits/9c6b3618db428edc90ebd42e1e28d77059560867